src/test/java/com/puppycrawl/tools/checkstyle/JavadocPropertiesGeneratorTest.java

```diff
@@ -78,73 +77,90 @@ protected final String getNonCompilablePath(String filename) throws IOException
 
     @Test
     public void testIsProperUtilsClass() throws ReflectiveOperationException {
-        assertTrue("Constructor is not private", TestUtil.isUtilsClassHasPrivateConstructor(
-            JavadocPropertiesGenerator.class, false));
+        assertTrue(TestUtil.isUtilsClassHasPrivateConstructor(
+            JavadocPropertiesGenerator.class, false), "Constructor is not private");
     }
 
     @Test
-    public void testNonExistentArgument() throws Exception {
+    public void testNonExistentArgument(@SysErr Capturable systemErr, @SysOut Capturable systemOut)
+            throws Exception {
+        systemErr.capture();
+        systemOut.capture();
```

Comment:
```
systemErr.capture();
        systemOut.capture();
```
looks like these lines are duplicated many times.
May be extract them to some method with @BeforeEach annotation?

Reply:
> May be extract them to some method with `@BeforeEach ` annotation?

It works! Thank you very much.